docs/installation/installation.md

```diff
@@ -10,8 +10,9 @@
   - [Watching Namespaces](#watching-namespaces)
   - [Watching custom ingress class](#watching-custom-ingress-class)
   - [Installing cert-manager](#installing-cert-manager)
```

Comment:
Should we move Installing cert-manager out of Tyk Operator Configuration?
I think it should be after/before Installing Tyk

Reply:
The section on installing cert-manager has been moved after the section on installing Tyk.